sdk/src/main/java/com/ifountain/opsgenie/client/model/BaseResponseWithId.java

```diff
@@ -1,13 +1,15 @@
 package com.ifountain.opsgenie.client.model;
 
+import com.ifountain.opsgenie.client.model.beans.BaseDataObj;
+
 public class BaseResponseWithId extends BaseResponse {
```

Comment:
Since this is no longer representing `Id`, this should be `BaseResponseWithData`